src/components/editor/GroupChoiceModal.jsx

```diff
@@ -28,18 +24,13 @@ const GroupChoiceModal = (props) => {
   }
 
   const saveAndClose = () => {
-    props.saveStarted()
-    const request = publishRDFResource(props.currentUser, props.rdf(), selectedValue)
+    props.publishMyResource(props.currentUser, selectedValue)
+
+    if (props.error) {
+      alert(props.error)
```

Comment:
`props.error` should be shown when rendering, not when doing `saveAndClose`, because otherwise it might not be show until the *next* time you call `saveAndClose`.

I think the standard way we've been showing errors is in an bootstrap styled alert div, not a browser alert.  Can you use that method for consistency?

Reply:
thanks - will do.